src/trellis/bundler/utils.py

```
@@ -0,0 +1,33 @@
+    Args:
+        tar: The tarfile to extract
+        dest: The destination directory
+
+    Raises:
+        ValueError: If any member would extract outside the destination
+    """
+    dest = dest.resolve()
+    for member in tar.getmembers():
+        member_path = (dest / member.name).resolve()
+        if not member_path.is_relative_to(dest):
+            raise ValueError(f"Tarball contains path traversal: {member.name}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)